### DISCUSSION TITLE
Plausible integration in Wowchemy hugo platform?

### DISCUSSION THREAD

**Discussion by mxmfrtn** [divinerites/plausible-hugo#16] [General]
Hello folks, 

I'm looking to use Plausible for my website analytics, but ran into a little problem to integrate the script. 

My website is deployed with Wowchemy on Netlify (Hugo). I can add the module imports and plausible params details in the config.toml file, but I don't know where to add the partial head HTML code. There are no index.html files anywhere in the file structure. 

If that's any help, here's the github repository for the Wowchemy Hugo academic theme: https://github.com/wowchemy/starter-hugo-academic The config.toml is under 

Am I missing something?

**Comment by divinerites** [comment]
This is purely a Wowchemy concern. Not a hugo or plausible-hugo concern.
Ask them how to add "something" in the header. Should be available.

The module you're calling from your theme is there, so the layouts : https://github.com/wowchemy/wowchemy-hugo-themes/tree/main/modules/wowchemy/layouts

**Reply by mxmfrtn** [reply]
Thanks divinerites for the quick reply. I also reached out on the wowchemy discussion thread and will see if I hear back anything.
